save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/projectcollection/CollectionView.kt

```diff
@@ -79,10 +109,60 @@ class CollectionView : AbstractView<CreationViewProps, State>() {
                 }
             }
         },
+        isTransparentGrid = true,
         initialPageSize = 10,
         useServerPaging = false,
         usePageSelection = false,
     )
+    private val privateProjectsTable: FC<TableProps<ProjectDto>> = tableComponent(
```

Comment:
I guess public projects and private projects have many common things like columns etc. Are you sure you want to copy-paste instead of creating a wrapper function or smth like that?

Reply:
I need two different table objects here

Reply:
You can create some kind of factory that will create those objects

Reply:
Yes. I misunderstood you.

Reply:
Fixed